src/Language/Haskell/HGrep/Query.hs

```diff
@@ -48,6 +53,12 @@ findTypeUses name src =
         . _unloc . _ConDeclH98 . _4 . _PrefixCon . folded
         . _unloc . to (typeContains name)
 
+
+findImports :: [Char] -> ParsedSource -> [SearchResult]
+findImports name src =
+  matchDeclsHs src $
+     _ImportDecl . _2 . _unloc . to (cn name)
```

Comment:
Curious if this is matching on the _module_ being imported, or the list of _values_ being imported? (We want two prisms, one for each!)